Connect.php

```diff
@@ -13,7 +13,7 @@
 
 // Check connection
 if ($con->connect_error) {
-    die("Database Connection failed: " . $conn->connect_error);
+    die("Database Connection failed: " . $con->connect_error);
```

Comment:
@dzr201732120115 @Teecloudy 

It is interesting.  Thanks, @dzr201732120115 .   Definitely we should use `$con` instead of `$conn`.

BTW, this file is different from the file I use on our server.

```
      1 <?php
      2 session_start();
      3
      4
      5
      6 $con=mysqli_connect("localhost","account_name","account_password","lrr");
      7 // Check connection
      8 if (mysqli_connect_errno())
      9   {
     10   echo "Failed to connect to MySQL: " . mysqli_connect_error();
     11   }
     12 else
     13 {
     14 echo "Connected";
     15 }
```
I think @Teecloudy  has updated this file but I have not updated this file on the server.
Should I update Connect.php on the server, @Teecloudy ?

**Hui**

Reply:
@spm2020spring 

Yes the connect.php need to be updated with the same database details used in the Script.php line 13 to line 18.

**Ashly**